Form-Controls/index.html

```diff
@@ -1,27 +1,49 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-  </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <header>
+      <h1>Customers'T-shirt Pick</h1>
+    </header>
+    <main>
+      <form>
+      <div>
+        <label for="username"> Enter username:</label>
+        <input type= "text" id="username" name="username" placeholder= "Username" required  autocomplete="off"/><br><br>   
+      </div>
+      <div>
+        <label for="email">Enter your email:</label>
+        <input type="email" id="email" name="email" placeholder="Email" required autocomplete="off"/><br><br>
+      </div>
+      <div>
+        <label for="T-shirtcolour">Choose a colour:</label>
+        <select id="T-shirtcolour" name="T-shirtcolour">
+          <option value="Pink">Pink</option>
+          <option value="Blue<">Blue</option>
+          <option value="Red">Red</option>
+        </select><br><br>
+      </div>
+      <div>
+        <label for="T-shirtsize">Choose a size:</label>
+        <select id="T-shirtsize" name="T-shirtsize">
+          <option value="XS">XS</option>
+          <option value="S">S</option>
+          <option value="M">M</option>
+          <option value="L">L</option>
+          <option value="XL">XL</option>
+          <option value="XXL">XXL</option>
+        </select>
+      </div>
+    </form>

```

Comment:
One thing that is missing here, that you would expect on a plain HTML form, is a submit button. This will allow the user to submit the form once they have completed it. It is also when the browser will have a chance to check for the validity of the form - i.e. has the user entered things for all of the required fields?

You could use for example a [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) here.

Reply:
Thanks for adding in a `<button>` here! If you change the `type` attribute of this element to "submit" instead of "button" it will try and submit the form, and you'll see the other work that you've done on the inputs for this form take effect - it will validate the form for the user and give feedback if they have entered in something that you are not expecting.